api/internals/usecase/activity_usecase.go

```diff
@@ -15,10 +15,10 @@ type activityUseCase struct {
 type ActivityUseCase interface {
 	GetActivities(context.Context) ([]domain.Activity, error)
 	GetActivityByID(context.Context, string) (domain.Activity, error)
-	CreateActivity(context.Context, string, string, string, string) error
-	UpdateActivity(context.Context, string, string, string, string, string) error
+	CreateActivity(context.Context, string, string, string, string) (domain.Activity, error)
+	UpdateActivity(context.Context, string, string, string, string, string) (domain.Activity, error)
 	DestroyActivity(context.Context, string) error
-	GetActivitiesWithSponsorAndStyle(context.Context) ([]domain.ActivityForAdminView, error)
+	GetActivitiesWithSponsorAndStyle(context.Context) ([]domain.ActivityDetail, error)
```

Comment:
```suggestion
	GetActivityDetail(context.Context) ([]domain.ActivityDetail, error)
```
ここも関数名長いので修正しちゃいましょう！(藤崎の過去技術的負債です...)
そうすると137行目の関数名も`GetActivityDetail`になります。
147行目の`AllWithSponsor`はrepositoryで定義した`FindDetail `になります。

また、controllerのメソッドも変更してほしいです。
IndexActivityWithSponsorAndStyle → `IndexActivityDetail`